test/integration/integration_suite_test.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/gengo/args"
+	"k8s.io/kube-openapi/pkg/generators"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+var package_name = "v1alpha1"
+var struct_name = "Model"
+
+var output_package = "main"
+var generated_base_name = "openapi_generated"
+var generated_file_name = generated_base_name + ".go"
+
+func TestGenerators(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Generators Suite")
+}
+
+var _ = Describe("Open API Definitions Generation", func() {
+
+	var tmpDir string
+
+	BeforeSuite(func() {
+
+		// Create the temporary working directory.
+		var err error
+		tmpDir, err = ioutil.TempDir("", "openapi")
+		if err != nil {
+			Expect(err).NotTo(HaveOccurred())
+		}
+
+		arguments := args.Default()
+
+		arguments.OutputBase = tmpDir
+		arguments.OutputPackagePath = output_package
+		arguments.InputDirs = []string{"./testdata"}
+		arguments.OutputFileBaseName = "openapi_generated"
+		// FIXME: can not depend on kubernetes code hierarchy
+		arguments.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt")
+
+		// Generate the OpenAPI definitions.
+		if err := arguments.Execute(
+			generators.NameSystems(),
+			generators.DefaultNameSystem(),
+			generators.Packages,
+		); err != nil {
+			glog.Fatalf("Error: %v", err)
+		}
+		glog.V(2).Info("Code for OpenAPI definitions generated")
+	})
+
+	AfterSuite(func() {
+		// os.RemoveAll(tmpDir)
+		gexec.CleanupBuildArtifacts()
+	})
+
+	Describe("Generating OpenAPI Definitions", func() {
+		Context("Open API definitions", func() {
+			It("should generate JSON swagger spec", func() {
+
+				// Open generated file for appending.
+				full_gen_file_path := filepath.Join(tmpDir, output_package, generated_file_name)
+				// FIXME: Change this to glog statement
+				fmt.Printf("File: %s\n", full_gen_file_path)
+				f, err := os.OpenFile(full_gen_file_path, os.O_APPEND|os.O_WRONLY, 0666)
+				defer f.Close()
+				Expect(err).ShouldNot(HaveOccurred())
+
+				// Append the test driver file to the generated file.
```

Comment:
ah now I understand. Can you just make the build command accept both files as input?